api_validation/pom.xml

```diff
@@ -27,6 +27,30 @@
     <artifactId>rapids-4-spark-api-validation</artifactId>
     <version>0.2.0-SNAPSHOT</version>
 
+    <properties>
+        <spark.version>3.0.0</spark.version>
+    </properties>
+    <profiles>
+        <profile>
+            <id>spark300</id>
+            <properties>
+                <spark.version>3.0.0</spark.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>spark301</id>
+            <properties>
+                <spark.version>3.0.1-SNAPSHOT</spark.version>
```

Comment:
we have these copied now in a couple places, perhaps we can put a <spark301.version> property in the top level pom file and then just reference it, that way when we change from snapshot to release we only need to change in one place. Same for 3.1.0-SNAPSHOT

Reply:
Thanks! Created PR https://github.com/NVIDIA/spark-rapids/pull/549 for above.